core/domain/role_domain.py

```diff
@@ -0,0 +1,76 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""contains required constants and functions for user roles and actions"""
+
+# dict to contain the hierarchy information for a role in oppia
+# NOTE FOR DEVELOPERS :
+# - Follow the Playbook in wiki before making any changes to this dict.
+# - Maintain the alphabetical in keys.
+ROLE_HIERARCHY = {
```

Comment:
done.

I think HIERARCHY is a better term as roles are not inherited, actions are. What do you think? Other terms that could be used may be ROLE_STRUCTURE and ACTION_INHERITANCE.